docs/release-notes/NuGet-5.5.md

```diff
@@ -0,0 +1,168 @@
+---
+title: NuGet 5.5 Release Notes
+description: Release notes for NuGet 5.5 including new features, bug fixes, and DCRs.
+author: karann-msft
+ms.author: karann
+ms.date: 03/19/2020
+ms.topic: conceptual
+---
+
+# NuGet 5.5 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version| Available in .NET SDK(s)|
+|:---|:---|:---|
+| [**5.5.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.5](https://visualstudio.microsoft.com/downloads/) | [3.1.200](https://dotnet.microsoft.com/download/dotnet-core/3.1)<sup>1</sup> |
+
+<sup>1</sup>Installed with Visual Studio 2019 with .NET Core workload
+
+## Summary: What's New in 5.5
+
+* Several accessibility fixes in PM UI
+
+* --skip-duplicate support for dotnet nuget push
+
+* msbuild /restore works with Packages.Config
+
+* Change the way we read XML files
```

Comment:
I'd add the `dotnet.exe reliability fixes with cross platform authentication plugins` in this list.